x-pack/plugins/security_solution/public/risk_score/containers/kpi/index.tsx

```diff
@@ -91,14 +91,13 @@ export const useUserRiskScoreKpi = ({
 }: UseUserRiskScoreKpiProps): RiskScoreKpi => {
   const spaceId = useSpaceId();
   const defaultIndex = spaceId ? getUserRiskIndex(spaceId) : undefined;
-  const usersFeatureEnabled = useIsExperimentalFeatureEnabled('usersEnabled');
 
   return useRiskScoreKpi({
     filterQuery,
     skip,
     defaultIndex,
     aggBy: 'user.name',
-    featureEnabled: usersFeatureEnabled,
+    featureEnabled: true,
```

Comment:
shouldn't we check `riskyUsersEnabled` here? 🤔

Reply:
Yes, it makes sense. I am changing it.